.buildkite/nightly-benchmarks/serving-tests.json

```diff
@@ -0,0 +1,70 @@
+[
+    {
+        "test_name": "serving_llama8B_tp1_sharegpt",
+        "server_parameters": {
+            "model": "meta-llama/Meta-Llama-3-8B",
+            "tensor_parallel_size": 1,
+            "swap_space": 16,
+            "disable_log_stats": "",
+            "disable_log_requests": "",
+            "load_format": "dummy"
+        },
+        "client_parameters": {
+            "model": "meta-llama/Meta-Llama-3-8B",
+            "dataset_name": "sharegpt",
+            "dataset_path": "./ShareGPT_V3_unfiltered_cleaned_split.json",
```

Comment:
We can also run the benchmark on sonnet dataset so that we can run benchmarks on different input, output, and prefix shapes. This is can be a TODO for later.

Reply:
I will put it as a TODO for later --- currently the benchmark already takes 40 minutes to run, and ideally I want to keep it <1hr